src/tasks/FishingTask.py

```
@@ -0,0 +1,581 @@
+                win32gui.ShowWindow(hwnd, win32con.SW_RESTORE)
+            except Exception:
+                pass
+            try:
+                foreground = win32gui.GetForegroundWindow()
+                if foreground != hwnd:
+                    current_tid = win32api.GetCurrentThreadId()
+                    target_tid = win32process.GetWindowThreadProcessId(hwnd)[0]
+                    win32process.AttachThreadInput(current_tid, target_tid, True)
+                    win32gui.SetForegroundWindow(hwnd)
+                    win32process.AttachThreadInput(current_tid, target_tid, False)
+            except Exception:
```

Ruff (0.15.11)

[warning] Do not catch blind exception: `Exception`

(BLE001)